src/code_80091750.c

```diff
@@ -12581,156 +12581,116 @@ void func_800A70E8(void *arg0) {
 GLOBAL_ASM("asm/non_matchings/code_80091750/func_800A70E8.s")
 #endif
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-s32 draw_box(s32, ?, ?, ?, s32, s32, s32, s32, s32); // extern
-extern s32 gDisplayListHead;
-
-void func_800A7258(void *arg0) {
-    s32 phi_v0;
-
+// Shading layer of the grand prix podium result screen
```

Comment:
Text box transparent background layer?

Reply:
Not a text box, just a regular box with a low alpha value filled in with black. Makes the screen look darker. It fades in prior to any actual text showing up